docker-compose.yml

```
@@ -0,0 +1,14 @@
+version: '3.0'
```

```suggestion
version: '3.8'
```

Let's pin to the latest version.

---

I haven't tested if this docker-compose is compatible with v3.8 compose file.
these versions are for the `docker-compose.yml` file, across the versions some add more features at the cost of mandatory complexity.

We can stick with v3.0 of docker-compose file until the end of days (caveat: if we do not need any functionality from greater versions, and do not mind slight change of syntax).

https://docs.docker.com/compose/compose-file/